confdb/confdb.go

```diff
@@ -626,12 +627,15 @@ type accessor interface {
 	keyType() keyType
 }
 
-func splitViewPath(path string) ([]string, error) {
+func splitViewPath(path string, opts parseOpts) ([]string, error) {
 	var subkeys []string
 	sb := &strings.Builder{}
 
 	finishSubkey := func() error {
 		if sb.Len() == 0 {
+			if opts.allowPartialPath {
```

Comment:
this seems to allow .. in the middle as it's allowed also in the loop ? are we checking that somewhere else? or yet something else